### PR TITLE
Bug 1268891 - pods not always grouped when service selector should cover template of a dc/deployment

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt would like you to look at this one

File: assets/app/scripts/controllers/overview.js [hunk 1]
```diff
@@ -227,7 +227,7 @@ angular.module('openshiftConsole')
       $scope.deploymentConfigsByService = {"": {}};
       angular.forEach($scope.deploymentConfigs, function(deploymentConfig, depName){
         var foundMatch = false;
-        var depConfigSelector = new LabelSelector(deploymentConfig.spec.selector);
+        var depConfigSelector = new LabelSelector(deploymentConfig.spec.template.metadata.labels);
```
Comment (liggitt) [inline assets/app/scripts/controllers/overview.js]: guard

**Comment by liggitt** [comment]
nit, then LGTM

**Comment by jwforres** [comment]
[merge]

**Pull request merged**